public/locales/en/panel-navigation.json

```diff
@@ -25,10 +25,10 @@
     },
     "focus-view": {
       "title": "Focus",
-      "info": {
-        "part-1": "Click the <focusIcon /> button to focus/retarget object.",
-        "part-2": "- Hold <keybind>Shift</keybind> when clicking to set as focus/anchor without retargetting.",
-        "part-3": "- Hold <keybind>Ctrl</keybind> when clicking to keep current camera velocities."
+      "info":  {
+        "click-focus-button": "Click the <focusIcon /> button to focus/retarget object.",
```

Comment:
Can you put a link into the readme explaining the syntax for this?  For example `<focusIcon />` not sure

Reply:
👍 we should probably add examples of some commonly used methods for localization in the readme, in general. I can see that it would not be obvious to the translator that these tags needs to be preserved when translating

Here's how it's used in code. `focusIcon` is a named tag used here to insert the icon into the text
![image](https://github.com/user-attachments/assets/4bc25609-5a6b-47a9-b7a6-3505410fc71f)

I feel like we might want to avoid these types of cases as much a possible, for simpler translation, but it would also be a bummer if we can't do fancy things in the text at all (andreas and I discussed another situation yesterday, where making part of a text string italic proved to be quite difficult/unintuitive)

Reply:
Doing it sparingly, but having examples is a good middle-ground, I agree